rbac/management/audit_log/model.py

```diff
@@ -80,9 +80,8 @@ def get_resource_item(self, r_type, request, *args, **kwargs):
             group_object_name = "group: " + group_object.name
             return group_object_id, group_object_name
 
-        elif r_type == AuditLog.PERMISSION:
-            # TODO: update for permission related items
-            return None
+        else:
+            return TypeError("Wrong Resource Type")
```

Comment:
I would prefer here the **ValueError** because we got an unexpected value of the resource type, the TypeError is more suitable for values that are not correct type, e.g. you expected a number and got a string